### ISSUE TITLE
SALT2 dispersion model

### ISSUE THREAD

**Issue by kbarbary** [sncosmo/sncosmo#6]
The SALT2 model for describing dispersion in light curves has not been
implemented.

In ``Model.bandflux()``, the keyword argument ``include_error=True`` results in the model uncertainty being returned in addition to the bandflux. However, this is *not* the SALT2 light curve dispersion (rather, it is a somewhat arbitrarily-set relative error that depends on phase and band wavelength).

**Comment by kbarbary** [comment]
I think we've got this. Still would like to put a unit test in, but I'm going to open a separate issue for that. Running snfit and figuring out exactly what it assuming is turning out to be harder than expected.
